src/executables/server.c

```diff
@@ -220,7 +220,7 @@ crash_handler (int signo, siginfo_t * siginfo, void *dummyp)
       return;
     }
 
-  if (!BO_IS_SERVER_RESTARTED () || !prm_get_bool_value (PRM_ID_AUTO_RESTART_SERVER))
+  if (!BO_IS_SERVER_RESTARTED ())
```

Comment:
더이상 crash_handler가 동작하면 안됩니다. 다음은 crash_handler에 의해 cub_server가 되살아난 결과입니다. 참고로, crash_handler는 release 빌드에서만 동작합니다. 시그널 핸들러에서 서버를 되살리는 행위 자체를 제거해야 하겠습니다. (확인 시 debug 모드 포함해서 검토)
```
$ cubrid server start demodb
@ cubrid master start
[SERVER_REVIVE_DEBUG] : server_entry_map is created.
[SERVER_REVIVE_DEBUG] : server_monitor_thread is created.
++ cubrid master start: success
@ cubrid server start: demodb

This may take a long time depending on the amount of recovery works to do.

CUBRID 11.4

[SERVER_REVIVE_DEBUG] : server entry has been registered into master_Server_monitor : pid : 43434, exec_path : /home/hornetmj/CUBRID/bin/cub_server, args : cub_server demodb
++ cubrid server start: success
Calling java stored procedure is allowed
hornetmj5: [~]
$ psc
hornetmj  43431      1  0 18:49 pts/5    00:00:00 cub_master
hornetmj  43434      1 10 18:49 ?        00:00:00 cub_server demodb
hornetmj  43656      1 14 18:49 ?        00:00:00 cub_javasp demodb
hornetmj5: [~]
$ kill -SIGABRT 43434
hornetmj5: [~]
$ psc
hornetmj  43431      1  0 18:49 pts/5    00:00:00 cub_master
hornetmj  43434      1  2 18:49 ?        00:00:00 cub_server demodb
hornetmj  43656      1  1 18:49 ?        00:00:00 cub_javasp demodb
hornetmj  43744  43434  0 18:49 ?        00:00:00 cub_server demodb
```

Reply:
`crash_handler`에서 server 되살리는 동작을 제거하였습니다.